Appraisals

```diff
@@ -1,11 +1,11 @@
+appraise 'activerecord-5.2' do
+  gem 'activerecord', '5.2.0'
```

Comment:
Why not using a version like `~> 5.2.0`?

Reply:
Also, why specifically 5.2 and not 5.0? In the Gemspec you specified 5.0, I suggest we stick with that version.

Reply:
I don't get that. This is PR to add support for 5.2, why would I specify 5.0 in 5.2 appraisal?

Reply:
Well, we're dropping support for 4.2, it does not mean that we drop support for 5.0 I guess, correct?

Reply:
Correct, that's why Appraisal file has definitions for 5.0, 5.1 and now for 5.2